presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -81,6 +81,7 @@
     public static final String TASK_PARTITIONED_WRITER_COUNT = "task_partitioned_writer_count";
     public static final String TASK_CONCURRENCY = "task_concurrency";
     public static final String TASK_SHARE_INDEX_LOADING = "task_share_index_loading";
+    public static final String TRACK_MEMORY_IN_TABLE_FINISH_OPERATOR = "track_memory_in_table_finish_operator";
```

Comment:
Ideally we shouldn't allow users disable memory tracking. Let's have it as a configuration property only.

Reply:
Good point, removed